### PR TITLE
Force Display Capture

### PR DESCRIPTION
**Pull request by Segergren**
* Added Force Display Capture to bypass the waiting time for Game Capture in unsupported games.
* Added CS2 to the list of detected games with Force Display Capture enabled.
* Removed SDL_APP from the list of blocked classes, as it was incorrectly included.
* Corrected the file path for "Detections" when running in debug mode.

Tested and verified with CS2 and other games that are compatible with Game Capture.

### CODE REVIEW

File: Resources/detections/gameDetections.json
Comment (lulzsun) [inline Resources/detections/gameDetections.json:1]: I feel that this makes a lot of unnecessary additions when only one game (Counter-Strike 2) only needs "force_display_capture" to be true. "force_display_capture" can be an optional property and you can do a null check or something similar to see if force_display_capture is true.

**Review by lulzsun — CHANGES_REQUESTED** [review]

File: Resources/detections/gameDetections.json
Comment (Segergren) [reply Resources/detections/gameDetections.json:1]: There is other games that doesn't work with Game Capture (like Destiny 2) but I added CS2 for now.
I don't understand what you mean by optional property. Do you mean that the user needs to turn on/off the property/setting every time when changing game?

Comment (lulzsun) [reply Resources/detections/gameDetections.json:1]: I meant that if a game needs "force_display_capture" to be true, then that property will be set in `gameDetections.json`, but if a game doesn't need it ("force_display_capture": false) then we shouldn't need to add that property to games that don't need it.

```diff
...
{
-	"force_display_capture": false,     <--- remove this line because it can be optional
	"region": null,
	"title": "FiveM",
	"game_detection": [
		{
			"gameexe": "/FiveM_b.*_GTAProcess\\.exe|/FiveM.app/cache/subprocess/FiveM_GTAProcess\\.exe",
			"longname": "FiveM"
		},
		{
			"gameexe": "/cache/subprocess/FiveM_GTAProcess\\.exe",
			"longname": "FiveM"
		}
	]
},
{
+	"force_display_capture": true,     <--- add this line because we need it
	"region": null,
	"title": "Counter-Strike 2",
	"game_detection": [
		{
			"gameexe": "cs2\\.exe$",
			"launchexe": "/steam\\.exe",
			"launchflags": "-applaunch 730",
			"regkey": "HKEY_LOCAL_MACHINE/SOFTWARE/Valve/Steam/InstallPath",
			"longname": "Counter-Strike 2 -Steam",
			"edition": "(Steam)"
		}
	]
},
...
```

And in `DetectionService.cs` we can do something like this:
```csharp
...
bool forceDisplayCapture = false;
if (gameDetectionsJson[x].TryGetProperty("force_display_capture", out JsonElement element)) {
    forceDisplayCapture = element.GetBoolean();
}
return (true, forceDisplayCapture, gameDetectionsJson[x].GetProperty("title").ToString());
...
```

Comment (Segergren) [reply Resources/detections/gameDetections.json:1]: Oh okay, that makes sense. Thanks for the feedback! Will fix it tomorrow.

Comment (Segergren) [reply Resources/detections/gameDetections.json:1]: Fixed in latest push 🫡

**Review by lulzsun — APPROVED** [review]
Looks great! Ready to merge, thank you!

**Pull request merged**